.github/workflows/build.yml

```diff
@@ -30,14 +30,69 @@ jobs:
           path: '.'
           pattern: '*.sh'
           exclude: './vendor/*'
+      
+      - name: License check
+        run: make license-check
   
-  jiva:
+  feature-test:
+    runs-on: ubuntu-latest
+    needs: ['lint']
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Build
+        run: make build
```

Comment:
Does this target uses the Dockerfile to build?

Reply:
Yes but that is not the buildx dockerfile and is used only for tests